app/models/katello/erratum.rb

```diff
@@ -52,8 +52,9 @@ def self.applicable_to_hosts(hosts)
     end
 
     def self.applicable_to_hosts_dashboard(hosts)
+      ids = hosts.to_sql.sub(/SELECT.*FROM/, "SELECT #{::Host.connection.quote_table_name('hosts')}.#{::Host.connection.quote_column_name('id')} FROM")
```

Comment:
I don't recall, either it hit the same issue or it still loaded all the columns instead of just the id.